as_perimenoun_oi_gynaikes.ass

```diff
@@ -775,7 +776,7 @@ Dialogue: 0,0:58:11.76,0:58:13.92,Default,,0,0,0,,No more bets please.
 Dialogue: 0,0:58:31.06,0:58:33.71,Default,,0,0,0,,Will you look at the imbecile betting on 36 red?
 Dialogue: 0,0:58:33.71,0:58:35.71,Default,,0,0,0,,Not even we could pull this off.
 Dialogue: 0,0:58:46.74,0:58:48.84,Default,,0,0,0,,36 red.
-Dialogue: 0,0:58:56.04,0:58:59.36,Default,,0,0,0,,That's motherfucking it! I'm voting for PASOK!
+Dialogue: 0,0:58:56.04,0:58:59.36,Default,,0,0,0,,What the fuck! I'm voting for PASOK!

```

Comment:
This line is iconic and we won't change it. That being said, your correction does not improve to the translation, it just is an alternative.